<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sonet_canvas</class>
 <widget class="QWidget" name="sonet_canvas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>800</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SONet Mars Mission Flight Sequence Viewer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QDockWidget" name="dockw_sc_info">
     <property name="minimumSize">
      <size>
       <width>119</width>
       <height>244</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>524287</width>
       <height>524287</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="windowTitle">
      <string>S/C Info</string>
     </property>
     <widget class="QWidget" name="dockw_sc_info_contentsw">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTreeWidget" name="treeW_sc_info_filter">
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDockWidget" name="dockw_trajectories_filter">
     <property name="minimumSize">
      <size>
       <width>95</width>
       <height>117</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>524287</width>
       <height>524287</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="windowTitle">
      <string>Trajectories Filter</string>
     </property>
     <widget class="QWidget" name="dockw_trajectories_contentsw">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTreeWidget" name="treeW_trajectories_filter">
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDockWidget" name="dockw_active_trips">
     <property name="maximumSize">
      <size>
       <width>524287</width>
       <height>150</height>
      </size>
     </property>
     <property name="allowedAreas">
      <set>Qt::AllDockWidgetAreas</set>
     </property>
     <property name="windowTitle">
      <string>Active Trips</string>
     </property>
     <widget class="QWidget" name="dockw_active_trips_contentsw">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTreeWidget" name="treeW_active_trips">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>150</height>
          </size>
         </property>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
